devolutions-gateway/src/api/jrec.rs

```diff
@@ -245,9 +255,9 @@ async fn jrec_delete_many(
     fn process_request(
         delete_list: Vec<Uuid>,
         recording_path: &Utf8Path,
-        active_recordings: &HashSet<Uuid>,
+        active_recordings: &ActiveRecordings,
     ) -> Result<ProcessResult, HttpError> {
-        let conflict = delete_list.iter().any(|id| active_recordings.contains(id));
+        let conflict = delete_list.iter().any(|id| active_recordings.contains(*id));
```

Comment:
Revert this too. It’s much less efficient as this version will need to acquire the lock for each value in the delete list.

Reply:
updated